_data/hipaa-apps.yml

```diff
@@ -0,0 +1,57 @@
+features:
+  - title: Service discovery
+    description: Apps must look up service endpoints via environment variables, or DNS, or specific IP addresses, or a client-library.
+    available: "yes"
+
```

Comment:
Oh, I didn't mean for this text to be used 100% verbatim. It was a Slack thread and not copywriting meant for a website 😁 

E.g., This should talk about what service discovery is. The "Service Resilience" item below shouldn't say "A ton of other concerns." The logging item says, "Apps that are Dockerized should be able to “just” send," with the "just" responding to a comment from Yori in that Slack thread. 

So all this text needs clean up / polish.